src/docfx/SubCommands/DocumentBuilderWrapper.cs

```diff
@@ -190,9 +190,12 @@ private static IEnumerable<Assembly> LoadPluginAssemblies(string pluginDirectory
 
         private static IEnumerable<DocumentBuildParameters> ConfigToParameter(BuildJsonConfig config, TemplateManager templateManager, ChangeList changeList, string baseDirectory, string outputDirectory, string templateDir)
         {
-            var parameters = new DocumentBuildParameters();
-            parameters.OutputBaseDir = outputDirectory;
-            parameters.ForceRebuild = config.Force ?? false;
+            var parameters = new DocumentBuildParameters
```

Comment:
nice, thanks

---
In reply to: [94204965](https://github.com/dotnet/docfx/pull/1091#discussion_r94204965) [](ancestors = 94204965,94204176)